cmd/internal/cli/actions.go

```diff
@@ -200,15 +201,6 @@ func replaceURIWithImage(ctx context.Context, cmd *cobra.Command, args []string)
 		return origImageURI
 	}
 
-	if err != nil {
```

Comment:
Don't think this should be dropped?

It is handling where one of the `case`s returns an error which is not a OCI-SIF mount issue.

Reply:
Yeah, not sure how that ended up being part of what I removed...